ui/views.py

```diff
@@ -227,6 +227,10 @@ def dispatch(self, *args, **kwargs):
     def extra_context(self):
         """Add to the context."""
         context = super(RepositoryView, self).extra_context()
+        vocabularies = [
+            x.replace(" ", "_")
```

Comment:
It's only done here to match the other one.

The trivial solution would be to disallow spaces in vocabulary names.